src/App/component/AddRepositoryDialog.jsx

```diff
@@ -37,23 +54,21 @@ export default function AddRepositoryDialog({ open, reloadProjects, handleClose,
       checker.push(checkRepositoryURL(repositoryURL));
 
       Promise.all(checker)
```

Comment:
Suggest not use both `.then` and `async await`.
```suggestion
      const urlCheckResults = await Promise.all(checker)
```

Reply:
Hope the next team can improve this comment XDD.